### PR TITLE
dev: added missing .github ci part

### PR DESCRIPTION
**Pull request by tg-bot-gurus**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    needs: run-tests
+    name: "Release"
+    runs-on: ubuntu-20.04
+    permissions: write-all
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: npm install @semantic-release/exec@6.0.3 -D
```
Comment (bissquit) [inline .github/workflows/release.yml]: Try to install using action in the next task

File: chart/values.yaml [hunk 2]
```diff
@@ -6,10 +6,10 @@ customCaCertUsed: false
 caCertSecretName: &caCertSecretName "cluster-ca-secret"
```
Comment (bissquit) [inline chart/values.yaml]: Yaml anchors look pretty good, but in case of Helm the most suitable tool to reuse code is `_helpers.tpl` file

**Review by bissquit — CHANGES_REQUESTED** [review]

File: chart/templates/_helpers.tpl [hunk 3]
```diff
@@ -1,25 +1,15 @@
...
-    https://github.com/helm/helm/issues/1054
-    https://github.com/helm/helm/issues/3684
-    https://helm.sh/docs/chart_template_guide/variables/
-*/}}
-
-{{- define "common.labels" -}}
-helm.sh/chart: {{ .Name }}-{{ .Version | replace "+" "_" }}
+{{/* Common labels */}}
+{{- define "common.labels" }}
+  labels:
```
Comment (AleksandrTitov) [inline chart/templates/_helpers.tpl]: You don't need to set"labels:" here.

The best approach is to define list of labels here and add it to resources using the following construction:

```
  labels:
    {{- include "common.labels" $ | nindent 4 }}
```

Also, add common labels to all the resources of the Helm chart

File: chart/templates/certificate.yaml [hunk 4]
```diff
@@ -3,22 +3,22 @@
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
-  name: {{ .Values.caCertificate.name | quote }}
-  namespace: {{ .Release.Namespace | quote }}
+  {{- with .Values.caCertificate }}
+  name: {{ .name | quote }}
+  namespace: {{ $.Release.Namespace | quote }}
```
Comment (AleksandrTitov) [inline chart/templates/certificate.yaml]: It's a value by default, but if you want, you can keep it

File: .github/workflows/release.yml (hunk 1 above)
Comment (tg-bot-gurus) [reply .github/workflows/release.yml]: @bissquit , is [this](https://github.com/cloud-labs-infra/helm-certificate-issuers/pull/1/commits/5dce8df13b61857d2bf492cd5ddc8935576c545d) smth you are looking for?

File: chart/templates/secret.yaml [hunk 5]
```diff
@@ -5,7 +5,7 @@ kind: Secret
 metadata:
   name: {{ .Values.caCertSecretName | quote }}
   namespace: {{ .Release.Namespace | quote }}
```
Comment (AleksandrTitov) [inline chart/templates/secret.yaml]: The same

**Review by bissquit — APPROVED** [review]

**Pull request merged**